src/lib/ng-v5/init/init-tsconfig.transform.ts

```diff
@@ -1,12 +1,19 @@
 import { ParsedConfiguration } from '@angular/compiler-cli';
+import chalk from 'chalk';
 import { Transform, transformFromPromise } from '../../brocc/transform';
 import { isEntryPoint, EntryPointNode } from '../nodes';
 import { initializeTsConfig } from '../../ts/tsconfig';
+import { msg } from '../../util/log';
 
 export const initTsConfigTransformFactory = (defaultTsConfig: ParsedConfiguration): Transform =>
   transformFromPromise(async graph => {
     // Initialize tsconfig for each entry point
     const entryPoints = graph.filter(isEntryPoint) as EntryPointNode[];
     initializeTsConfig(defaultTsConfig, entryPoints);
+
+    if (defaultTsConfig.options.enableIvy) {
+      msg(chalk.yellow('\n** It is not recommended to publish Ivy libraries to NPM repositories **'));
```

Comment:
I would recommend we provide some more context here, and maybe use stronger wording than "not recommended". I think it's important to communicate what the consequences of publishing would be for the library. Maybe just link to an MD file in the repo with additional info, similar to schematics documentation.

Reply:
Probably make sense to have something on AIO, eventually.